package.json

```diff
@@ -13,7 +13,6 @@
     "postsemantic-release": "npm run coverage"
   },
   "nyc": {
-    "//": "don't let nyc instrument users-design-doc.js because it will be executed inside PouchDB",
```

Comment:
@gr2m Gregor, I noticed that green keeper removed this comment... I guess because I added two "//" keys. I think the first comment is more important than the second, so if it's ok, I will create a new pull request to improve the documentation in package.json. Does that sound OK?